Sources/OAuth2Client/NXOAuth2AccountStore.m

```diff
@@ -545,12 +547,8 @@ - (void)oauthClientDidRefreshAccessToken:(NXOAuth2Client *)client
     }
     
     foundAccount.accessToken = client.accessToken;
-    NSDictionary *userInfo = [NSDictionary dictionaryWithObject: foundAccount
-                                                         forKey: NXOAuth2AccountStoreNewAccountUserInfoKey];
-    
-    [[NSNotificationCenter defaultCenter] postNotificationName:NXOAuth2AccountStoreAccountsDidChangeNotification
-                                                        object:self
-                                                      userInfo:userInfo];
+    [[NSNotificationCenter defaultCenter] postNotificationName:NXOAuth2AccountDidChangeAccessTokenNotification
```

Comment:
Why did you change the notification. This is part of the account store and `NXOAuth2AccountStoreAccountsDidChangeNotification` is the correct notification type. In any case this is not part of the crash fix.